configs/ext/crypto_config_profile_medium.h

```diff
@@ -243,7 +243,7 @@
 /**
  * \def MBEDTLS_ENTROPY_C
  *
- * Enable the platform-specific entropy code.
+ * Enable the generic entropy code.
```

Comment:
To my understanding ext files are consumed and by external projects, in that case TF-M?

Would it be good to add a specific changelog text to communicate that this file has been modified?

Reply:
It's the other way round: in principle, this file is a copy of the one in TF-M. But when we make incompatible changes to the library, we anticipate the changes that TF-M are likely to make.

TF-M are already aware of this pull request, and I'll notify them once it's merged.